tools/sidekick/config.json

```diff
@@ -1,6 +1,7 @@
 {
   "project": "Milo",
   "host": "milo.adobe.com",
+  "trustedHosts": ["milo.adobe.com", "*--milo--adobecom.aem.page"],
```

Comment:
@andreituicu, we need `*--milo--adobecom.aem.live` as well to support https://github.com/adobecom/milo/blob/stage/tools/sidekick/config.json#L108-L113.

Reply:
Sure! Normally, relative paths should work OOTB, because the current site is always a trusted host, but if this repo serves as some kind of template, then it makes sense to have this too.